Editor/UI/ProjectAuditorWindow.cs

```diff
@@ -22,20 +22,12 @@ private IssueTable m_ActiveIssueTable
         }
         
         private string[] m_AssemblyNames;
-
-        private enum AssemblyIndex
-        {
-            None = -1,
-            All = 0
-        }
-       
+        private TreeViewSelection m_AssemblySelection = null;
         private const string m_DefaultAssemblyName = "Assembly-CSharp";
         
         private SearchField m_SearchField;
         
         [SerializeField] private bool m_ValidReport = false;
-        [SerializeField] private int m_ActiveAssembly = (int)AssemblyIndex.None;
```

Comment:
I believe with this PR we lose the selection on domain reload. Previously, to preserve the selection we had m_ActiveAssembly/m_ActiveArea tagged with SerializeField. Perhaps we should have something similar in AssemblySelectionWindow/AreaSelectionWindow.

Reply:
Good spot. Fixed. It's not possible to directly serialize the TreeViewSelection, but since we generate a string with a summary of the selection it's possible to store and serialize that, then parse it to reconstruct the TreeViewSelection